desktop/flipper-ui-core/src/fb-stubs/pluginMarketplaceAPI.tsx

```diff
@@ -7,10 +7,10 @@
  * @format
  */
 
-export async function loadPluginsFromMarketplace() {
-  // Marketplace is not implemented in public version of Flipper
-}
+import {MarketplacePluginDetails} from '../reducers/plugins';
 
-export default () => {
-  // Marketplace is not implemented in public version of Flipper
-};
+export async function loadAvailablePlugins(): Promise<
+  MarketplacePluginDetails[]
+> {
+  return [];
```

Comment:
This is the place where we can hook returning details of the plugins available for downloading.